Dockerfile

```diff
@@ -1,19 +1,19 @@
-FROM gradle:4.7.0-jdk8 as build
+FROM gradle:5.4.1-jdk11 as build
 
 WORKDIR /trust-anchor
 USER root
 ENV GRADLE_USER_HOME ~/.gradle
 
 COPY build.gradle build.gradle
-RUN gradle install
+RUN gradle installDist
```

Comment:
I'm probably missing something but why is `gradle installDist` called twice?

Reply:
Good spot, no this is a mistake